lib/internal/Magento/Framework/HTTP/PhpEnvironment/RemoteAddress.php

```diff
@@ -139,7 +144,7 @@ public function getRemoteAddress(bool $ipToLong = false)
 
         $this->remoteAddress = $remoteAddress;
 
-        return $ipToLong ? ip2long($this->remoteAddress) : $this->remoteAddress;
+        return $ipToLong && $this->remoteAddress ? ip2long($this->remoteAddress) : $this->remoteAddress;
```

Comment:
Can we revert this? The goal on this one is literally just to mark classes/interfaces as API and not make any code change. If this is required because of `strict_types=1`, lets remove it and create a separate PR for it.